lib/web_of_science/process_records.rb

```diff
@@ -36,10 +36,10 @@ def execute
       # @return [Array<String>] WosUIDs that create a new Publication
       def create_publications
         filtered_records = filter_databases
```

Comment:
Would you be OK with skipping this unnecessary assignment and just using `filter_databases` as the param in the next line?

Reply:
This change could be made, but it's not essential to behavior, it's just a style preference AFAICT.

Reply:
Anyway, done.